cattenbak.py

```diff
@@ -13,8 +13,11 @@
 discovery_url = "https://discovery.eduroam.app/v1/discovery.json"
 
 
-def get_seq(old_discovery):
-    return int(old_discovery["seq"]) + 1
+def get_serial():
+    old_discovery = requests.get(discovery_url).json()
+    old_serial = int(old_discovery["serial"]) if "serial" in old_discovery else 0
+    candidate_serial = int(datetime.datetime.now().strftime("%Y%m%d00"));
+    return candidate_serial if candidate_serial > old_serial else old_serial + 1
```

Comment:
Works fine, but wouldn't seconds in strftime be even more simple?

Reply:
This is an alternative for `seq`; it's supposed to be reasonably short. The `Last-Modified` HTTP header can be used to obtain the date the file was last uploaded.